sdk/iot/core/src/az_iot_telemetry.c

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "az_iot_hub_client.h"
+#include <az_contract_internal.h>
+#include <az_result.h>
+#include <az_span.h>
```

Comment:
suggestion:
how about following  this guideline to arranging our header file.

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g. <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g. <algorithm>, <cstddef>.
A blank line
Other libraries' .h files.
Your project's .h files.



source:[https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
i just let clang_format do it's work. I also need `#include <stdint.h>`. I'll check it out